libkbfs/keybase_daemon_rpc.go

```diff
@@ -305,3 +306,46 @@ func (k *KeybaseDaemonRPC) Shutdown() {
 		k.shutdownFn()
 	}
 }
+
+// GetTLFCryptKeys implements the TlfKeysInterface interface for
+// KeybaseDaemonRPC.
+func (k *KeybaseDaemonRPC) GetTLFCryptKeys(
+	ctx context.Context, tlfName string) (res keybase1.TLFCryptKeys, err error) {
+	var tlfHandle *TlfHandle
+
+getHandle:
+	for {
+		tlfHandle, err = ParseTlfHandle(ctx, k.config.KBPKI(), tlfName, false)
+		switch e := err.(type) {
+		case TlfNameNotCanonical:
+			tlfName = e.NameToTry
+		case nil:
+			break getHandle
+		default:
+			return res, err
+		}
+	}
+
+	res.CanonicalName = keybase1.CanonicalTlfName(tlfHandle.GetCanonicalName())
+
+	var rmd ImmutableRootMetadata
+	var id TlfID
+	_, rmd, id, err = k.config.KBFSOps().GetOrInitializeNewMDMaster(
+		ctx, k.config.MDOps(), tlfHandle)
+	if err != nil {
+		return res, err
+	}
+	res.TlfID = keybase1.TLFID(id.String())
```

Comment:
I don't have an idea yet about how it'd be used; sorry! But I also wasn't sure it's not useful at all. So returning the TlfID in this RPC call is to have it (and not blocking other tickets) when it's needed, but make it trivial to remove if it turns out not useful. My understanding is if it becomes widely used in chat, this would be split into a separate `GetTlfID` call.